lib/trailblazer/developer/operation.rb

```diff
@@ -0,0 +1,68 @@
+module Trailblazer::Developer
+  # Operation-specific circuit rendering. This is optimized for a linear railway circuit.
+  #
+  # @private
+  #
+  # NOTE: this is absolutely to be considered as prototyping and acts more like a test helper ATM as
+  # Inspect is not a mission-critical part.
+  class Operation
+    def self.introspect(*args)
+      Inspect.(*args)
+    end
+
+    module Inspect
+      module_function
+
+      def call(operation, options = {style: :line})
+        graph = Trailblazer::Activity::Introspect::Graph(operation)
```

Comment:
The main problem here is that we are passing an operation but `Introspect::Graph` expects an `activity`.
How do we "transform" an operation into an activity? Is that possible?

Reply:
Of course, all TRB components have a decomposer called `#to_h`. Operations have the same interface than activities, that's why this works.